nestkernel/simulation_manager.cpp

```diff
@@ -666,6 +704,12 @@ nest::SimulationManager::call_update_()
 void
 nest::SimulationManager::update_connection_infrastructure( const thread tid )
 {
+#pragma omp barrier
```

Comment:
This barrier is only called once, right? So one extra barrier probably does not have a huge impact on simulation time. So probably not the same as above.

Reply:
Yes, it's not frequently called. Keep this barrier.